keras_cv/layers/preprocessing/random_brightness.py

```diff
@@ -0,0 +1,119 @@
+# Copyright 2022 The KerasCV Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import tensorflow as tf
+
+_SCALE_VALIDATION_ERROR = ('The `scale` should be number or a list of two numbers '
+                           'that ranged between [-1.0, 1.0]. ')
+
+
+class RandomBrightness(tf.keras.layers.Layer):
+  """Randomly adjust brightness for the a RGB image.
+
+  This layer will randomly increase/reduce the brightness for the input RGB image.
+  During inference time, the output will be identical to input. Call the layer with
+  training=True to adjust brightness of the input.
+
+  Note that same brightness adjustment will be apply to all the images in the same
+  batch.
+
+  Args:
+    scale: Float or a list/tuple of 2 floats between -1.0 and 1.0. The scale is
```

Comment:
should we expose image pixel boundaries as well?  I'm starting to think we should support all formats for pixel values (i..e [0,1], [-1, 1], [0, 255]).

This seems like a decision we should make and keep consistent across all of our layers

Reply:
For brightness/contrast and any color related augmentation, I feel we should stay with RGB color (0~255). The normalization of the input (for model) should happen after the color adjustment.

Reply:
I also think so, like there is a separate [rescaling layer](https://keras.io/api/layers/preprocessing_layers/image_preprocessing/rescaling/) that can be used for pixel normalization after all transformation.